### ISSUE TITLE
Packages cannot be installed from nuget.org V3 Source Location

### ISSUE THREAD

**Issue by SydneyhSmith** [OneGet/oneget#428]
Package installation is broken for nuget.org V3 (https://api.nuget.org/v3/index.json), the source location must be set for V2 (https://www.nuget.org/api/v2). 

Reproduction:
![image](https://user-images.githubusercontent.com/43417619/51776583-b84d9000-20ae-11e9-9bfb-cd3559998b08.png)

**Comment by bergmeister** [comment]
Yes, please. Also provide better guidance on how to use Packagemanagement and PowerShellGet together with Azure DevOps. The best guidance so far is this super old issue [here](https://github.com/MicrosoftDocs/PowerShell-Docs/issues/1200#issue-227526581) that also described that v3 does not work.

**Comment by dejoost** [comment]
We also need this fix for save-package and install-package cmds. We have feeds on MyGet and there the v2 feeds are not even exposing prereleases which makes the v2 workaround useless
